src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -363,58 +363,14 @@ def __call__(self, hidden_state):
         return nn.tanh(out)
 
 
-class FlaxBertModule(nn.Module):
-    vocab_size: int
-    hidden_size: int
-    type_vocab_size: int
-    max_length: int
-    num_encoder_layers: int
-    num_heads: int
-    head_size: int
-    intermediate_size: int
-    dropout_rate: float = 0.0
-    kernel_init_scale: float = 0.2
-    dtype: jnp.dtype = jnp.float32  # the dtype of the computation
-
-    @nn.compact
-    def __call__(self, input_ids, attention_mask, token_type_ids, position_ids, deterministic: bool = True):
-
-        # Embedding
-        embeddings = FlaxBertEmbeddings(
-            self.vocab_size,
-            self.hidden_size,
-            self.type_vocab_size,
-            self.max_length,
-            kernel_init_scale=self.kernel_init_scale,
-            dropout_rate=self.dropout_rate,
-            name="embeddings",
-            dtype=self.dtype,
-        )(input_ids, token_type_ids, position_ids, attention_mask, deterministic=deterministic)
-
-        # N stacked encoding layers
-        encoder = FlaxBertEncoder(
-            self.num_encoder_layers,
-            self.num_heads,
-            self.head_size,
-            self.intermediate_size,
-            kernel_init_scale=self.kernel_init_scale,
-            dropout_rate=self.dropout_rate,
-            name="encoder",
-            dtype=self.dtype,
-        )(embeddings, attention_mask, deterministic=deterministic)
-
-        pooled = FlaxBertPooler(kernel_init_scale=self.kernel_init_scale, name="pooler", dtype=self.dtype)(encoder)
-        return encoder, pooled
-
-
 class FlaxBertPredictionHeadTransform(nn.Module):
     dtype: jnp.dtype = jnp.float32
 
     @nn.compact
     def __call__(self, hidden_states):
         hidden_states = nn.Dense(hidden_states.shape[-1], name="dense", dtype=self.dtype)(hidden_states)
-        hidden_states = nn.elu(hidden_states)  # TODO: ACT2FN[config.hidden_act]
-        return FlaxBertLayerNorm(name="LayerNorm", dtype=self.dtype)(hidden_states)
+        hidden_states = gelu(hidden_states)  # TODO: ACT2FN[config.hidden_act]
+        return FlaxBertLayerNorm(name="layer_norm", dtype=self.dtype)(hidden_states)
```

Comment:
Shouldn't this need to be handled also when coming from PyTorch side? PyTorch version LN layers were still stored has `self.LayerNorm`

Reply:
Yeah, it might be better to keep the same name as in PT... currently there is a mix of `layer_norm` and `LayerNorm` in `modeling_flax_bert.py`. I'd suggest that in the next PR we look into how to maybe change the naming for both FlaxBERT and FlaxRoBERTa to align it better with PyTorch. I think it should be possible to have only a single general `convert_pt_to_flax` function that is the same for all FlaxModels by forcing the same names in PT and Flax. Let's maybe handle that in the next PR? cc @LysandreJik @sgugger

Reply:
Also, we can look into how to handle loading incomplete models in Flax in the next PR, *e.g.*, if only parts of the model are found in the corresponding PyTorch or Flax, serialized model file

Reply:
I slightly prefer `"layer_norm"` as a name but it would need some replacements in the automatic conversion function that may or may not be dangerous, so I can live with `"LayerNorm"`. The important bit is to always use the same name.

Reply:
Guess we should also check this in the next PR once we figured out the general design to convert PT to FX